pkg/reconciler/pipelinerun/affinity_assistant_test.go

```diff
@@ -240,23 +267,33 @@ func TestCreateAndDeleteOfAffinityAssistantPerWorkspace(t *testing.T) {
 		tc := tc
 		t.Run(tc.name, func(t *testing.T) {
 			ctx := context.Background()
+			kubeClientSet := fakek8s.NewSimpleClientset()
 			c := Reconciler{
-				KubeClientSet: fakek8s.NewSimpleClientset(),
+				KubeClientSet: kubeClientSet,
+				pvcHandler:    volumeclaim.NewPVCHandler(kubeClientSet, zap.NewExample().Sugar()),
 			}
 
-			err := c.createOrUpdateAffinityAssistantsPerAABehavior(ctx, tc.pr, aa.AffinityAssistantPerWorkspace)
+			err := c.createOrUpdateAffinityAssistantsAndPVCs(ctx, tc.pr, tc.aaBehavior)
 			if err != nil {
-				t.Errorf("unexpected error from createOrUpdateAffinityAssistantsPerWorkspace: %v", err)
+				t.Fatalf("unexpected error from createOrUpdateAffinityAssistantsPerWorkspace: %v", err)
 			}
 
 			// validate StatefulSets from Affinity Assistant
 			for i, w := range tc.pr.Spec.Workspaces {
 				if tc.expectStatefulSetSpec != nil {
-					expectAAName := getAffinityAssistantName(w.Name, tc.pr.Name)
+					expectAAName := GetAffinityAssistantName(w.Name, tc.pr.Name)
 					validateStatefulSetSpec(t, ctx, c, expectAAName, tc.expectStatefulSetSpec[i])
 				}
 			}
 
+			// validate PVCs from VolumeClaimTemplate
```

Comment:
It looks like this is just checking the PVC exists? is it expected that the return value from the client is ignored?